### DISCUSSION TITLE
Screen share feature doesn't work when I embed the plugnmeet content in an "<iframe>"

### DISCUSSION THREAD

**Discussion by ThilinaV98** [mynaparrot/plugNmeet-server#340] [Q&A] [answered]
When I try to embed the plugnmeet content in an "<iframe>" other all features work properly but the Screen share feature doesn't work.

**Comment by jibon57** [comment, marked as answer]
We don't support to use iframe. You can check docs: https://www.plugnmeet.org/docs/api/get-client-files

**Reply by ThilinaV98** [reply]
Thank you.
